src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -706,8 +712,20 @@ def __call__(
                     noise_pred_uncond, noise_pred_text = noise_pred.chunk(2)
                     noise_pred = noise_pred_uncond + guidance_scale * (noise_pred_text - noise_pred_uncond)
 
+                    # Based on 3.4. in https://arxiv.org/pdf/2305.08891.pdf
+                    std_text = torch.std(noise_pred_text)
```

Comment:
This is incorrect. The std needs to be calculated as a single real number per image, not per pixel. Change it to:
```
std_text = noise_pred_text.flatten(1).std(1)
std_pred = noise_pred.flatten(1).std(1)
```